.github/workflows/continuous-integration.yml

```diff
@@ -1,12 +1,18 @@
 
 name: "Continuous Integration"
```

Comment:
```suggestion
name: "CI"
```

![image](https://user-images.githubusercontent.com/2228672/95664384-a55f4580-0b47-11eb-9088-3dddc2a2a4f3.png)

long names as not displayed completely... Maybe also rename the workflow file to `ci.yml` then

Reply:
> long names as not displayed completely...

They are:

![Screenshot_2020-10-10 Migrate jobs away from Travis to Github Actions by greg0ire · Pull Request #4310 · doctrine dbal](https://user-images.githubusercontent.com/657779/95664477-72698180-0b48-11eb-891f-e3139c271245.png)

Reply:
![image](https://user-images.githubusercontent.com/2228672/95664613-de001e80-0b49-11eb-8a28-2148078e83f0.png)

not everywhere...

Reply:
Indeed. Hopefully Github will fix that in the future. In the meantime, I'd rather not use abbreviations, which would mean being inconsistent with workflows that come from `doctrine/.github` and don't use abbreviations either.